src/lib/holocene/accordion/accordion.svelte

```diff
@@ -61,6 +61,7 @@
           {title}
```

Comment:
| Before | After |
|-|-|
|<img width="1197" alt="Screenshot 2024-09-17 at 11 03 36 AM" src="https://github.com/user-attachments/assets/d3fe2c6a-b0f4-4db2-b4fc-4fad0ae50470">|<img width="1206" alt="Screenshot 2024-09-17 at 11 03 53 AM" src="https://github.com/user-attachments/assets/4dadc289-baaf-4fe4-bc60-310f6ad6e3b4">|

Might want to update the following ⬇️  in order to fix the icon and word wrapping ⬆️ 

```js
      <div class="justify-between flex w-full flex-row items-center gap-2">
        <h3 class="flex items-center gap-2">
```
and

```js
        <Icon class="m-2 shrink-0" name={open ? 'chevron-up' : 'chevron-down'} />
```

Reply:
👍